staging/dex-k8s-authenticator/html-templates/windows-tab.html

```diff
@@ -85,8 +85,14 @@ <h3>Run configuration commands</h3>
     <button class="btn" style="float:right" data-clipboard-snippet="">
       <img class="clippy" width="13" src="{{ .Web_Path_Prefix }}static/clippy.svg" alt="">
     </button>
+
+    {{ if .K8sCaPem }}
```

Comment:
is the whitespace ahead of the logic block right here? I take it it works either way? :thinking:

Reply:
I added it for readability. It doesn't make any difference in the rendering.